src/api/property/float.js

```diff
@@ -0,0 +1,6 @@
+export default {
+  default: 0,
+  serialize: String,
+  deserialize: parseFloat,
```

Comment:
As discussed IRL, I'll just keep these as is for now. The only difference is that once an attribute is set, it will always exist on the element, even if it is the default. If no default is specified and `undefined` is passed, then the attribute *should* be removed.